lib/node_modules/@stdlib/math/base/special/log2/package.json

```
@@ -62,5 +62,81 @@
     "base 2",
     "logarithm",
     "log"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "log2",
+      "alias": "log2",
+      "pkg_desc": "compute the binary logarithm",
+      "desc": "computes the binary logarithm",
+      "short_desc": "binary logarithm",
+      "parameters": [
+        {
+          "name": "x",
+          "desc": "input value",
+          "type": {
+            "javascript": "number",
+            "jsdoc": "number",
+            "c": "double",
+            "dtype": "float64"
+          },
+          "domain": [
+            {
+              "min": "0",
```

```suggestion
              "min": 0,
```

`0` is valid JSON so we can use the literal.